src/main/java/org/ohdsi/authenticator/service/AccessTokenResolver.java

```diff
@@ -0,0 +1,23 @@
+package org.ohdsi.authenticator.service;
+
+public class AccessTokenResolver {
```

Comment:
I do not see usage of the class

Reply:
this one is used int the node/portal. I have added unit test anyway for this.

Reply:
done